api/manifest/manifest_test.go

```diff
@@ -177,6 +177,37 @@ worker:
 	_assert(t, cases)
 }
 
+func TestGenerateDockerfileProcfile(t *testing.T) {
+	destDir := mkBuildDir(t, "../../examples/dockerfile-procfile")
+	defer os.RemoveAll(destDir)
+
+	Init(destDir)
+	m, _ := Read(destDir, defaultManifestFile)
+
+	Execer = func(bin string, args ...string) *exec.Cmd {
+		return exec.Command("true")
+	}
+
+	cases := Cases{
```

Comment:
Ok I think I see a way to make this more generic. I'll deal with the "unknown" type being detected and have it generate generic Docker assets. Then we can rebase this onto it and update the tests for unknown instead of sinatra.